mkgu_packaging/dicarlo/marques/marques_ringach2002.py

```diff
@@ -0,0 +1,93 @@
+
+import numpy as np
+from brainio_collection.packaging import package_data_assembly
+import scipy.io as sio
+from brainio_base.assemblies import DataAssembly
+
+DATA_DIR = '/braintree/data2/active/users/tmarques/bs_datasets/Ringach2002.mat'
+ASSEMBLY_NAME = 'shapley.Ringach2002'
+ORIENTATION_STIM_NAME = 'dicarlo.Marques2020_orientation'
+PROPERTY_NAMES = ['baseline', 'max_dc', 'min_dc', 'max_ac', 'modulation_ratio', 'circular_variance', 'bandwidth',
```

Comment:
what do DC and AC stand for? If there's a more descriptive name, I would favor using that

Reply:
AC and DC refer to alternate current and direct current (what AC/DC usually refer to in electronics). In electrophysiology, AC is a pretty standard way to refer to the variable component of the response (first harmonic of the Fourier transform) and DC is the mean response.